src/main/kotlin/me/partlysanestudios/partlysaneskies/features/chat/ChatTransformer.kt

```diff
@@ -0,0 +1,51 @@
+//
+// Written by DerGruenkohl
```

Comment:
Do you want your github name or your discord name? It can be either, just online name that is identifiable to you

Reply:
Id rather have it my Github name as thats the name I usually go by online